snprc_scheduler/webpack/prod.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     },
 
     output: {
-        path: path.resolve(__dirname, '../resources/web/snprc_scheduler/app/'),
+        path: path.resolve(__dirname, '../resources/web/snprc_scheduler/gen/app/'),
```

Comment:
same question here, any reason to leave in this /app subdir within /gen?

Reply:
I know even less about what goes on here so have no strong opinion one way or the other.